content/soda-gettingstarted/installation-using-ansible.md

```diff
@@ -241,51 +253,70 @@ Delete the Fileshare
 ```bash
 osdsctl fileshare delete <fileshare id>
 ```
-## SODA Dashboard UI
-<br />
+
+### SODA Dashboard UI
 
 SODA Dashboard UI is available at `http://{your_host_ip}:8088`, please login the dashboard using the default admin credentials: `admin/opensds@123.` Create `tenant`, `user`, and `profiles` as admin. Multi-Cloud service is also supported by dashboard.
-<br />
+
+### Please Note
+{{% notice info %}}
+**_To use the multicloud service, an AK/SK must be generated before anything else. To do this you can follow the steps below:_**  <br />
+1. Go to AK/SK Management<br />
+2. Click on Add AK/SK button.<br />
+3. Save the file (Do not forget to save this file and keep it safe.)<br />
+{{% /notice %}}
+
+{{% notice info %}}
+**_To use the Block and File service a profile must be created before anything else. To do this you can follow the steps below:_**  <br />
+1. Go to Profiles<br />
+2. Click on Create button.<br />
+3. Enter the details and select the Storage type as Block or File. Create one profile for Block type and one for File.<br />
```

Comment:
Please mention the profile names must be default_block or default_file for block and file, respectively

Reply:
Will make the change.